src/Modules/tvApi.js

```diff
@@ -0,0 +1,72 @@
+const myUrl = 'https://api.tvmaze.com/shows/';
+
+const container = document.getElementById('cardContainer');
+
+// const display = (json) => {
+//   document.body.appendChild(myImg)
+// }
+
+const createCard = (obj, json) => {
+  // Create a div element with class card
+  const card = document.createElement('div');
+  card.className = 'card';
+
+  // Create a h3 element with the name of the show
+  const title = document.createElement('h3');
+  title.textContent = obj.name;
```

Comment:
The title should use the name given in the JSON, not the object, we are not supposed to use hard-coded titles.

Reply:
- Refactored to use the name given in the JSON.